packages/evm/src/errors.ts

```diff
@@ -1,6 +1,6 @@
-export type ERROR = (typeof ERROR)[keyof typeof ERROR]
+export type EVMErrorType = (typeof EVMError)[keyof typeof EVMError]
 
-export const ERROR = {
+export const EVMError = {
```

Comment:
Could we just rename this to `EVMErrorMessages` or something similar?  That would quickly clarify the difference between this and the `EvmError` class below.

Reply:
I went with this for now, in a future followup I can look into Jochem's suggestion of adding these messages as a static field in the constructor class, but for now, I'd rather get this merged with just the name change to differentiate so that we're not piling too many changes into this one PR.